contact.html

```diff
@@ -22,23 +24,24 @@
 		</div>
 		<div class="contactdetails">
 			<div class="contacticons">
-			<div class="mail"><a href="mailto:nickclifford@gmail.com">
-				<img alt="Email" src="mail.png" width="75" height="75">
+			<div id="mail"><a href="mailto:nickclifford@gmail.com">
+				<img alt="Email" src="mail.png" >
 				</a>
 			</div>
-			<div class="phone"><a href="tel:07970740492">
-				<img alt="Phone" src="phone.png" width="75" height="75">
+			<div id="phone"><a href="tel:07970740492">
+				<img alt="Phone" src="phone.png" >
 				</a>
 			</div>
-			<div class="twitter"><a href="https://twitter.com/cliffdeford">
-				<img alt="Twitter" src="twitter.png" width="75" height="75">
+			<div id="twitter"><a href="https://twitter.com/cliffdeford">
+				<img alt="Twitter" src="twitter.png">
 				</a>
 			</div>
 		</div>
 			<div class="contactform">
 			<h3>Or send me a message</h3>
 
 			<div class="formcontainer">
+				<form action="/nonexistent.php">
 			    <label for="fname">First Name</label>
 			    <input type="text" id="fname" name="firstname" placeholder="Your first name..">
 
```

Comment:
I can't comment below for some reason but footers should go within the body rather than after it. Also, you could 
write it as 
``` 
<footer>
          <p>© Nick Clifford 2021</p>
</footer>
```
to have greater semantic meaning. If you want to keep it as is, you should indent it so it's 
```
<div class="footer">
    	<p>© Nick Clifford 2021</p>
</div>
```